src/libos/crates/async-io/src/fs/stat_buf.rs

```diff
@@ -140,6 +140,7 @@ impl StatMode {
             FileType::BlockDevice => StatMode::BLOCK,
             FileType::Socket => StatMode::SOCKET,
             FileType::NamedPipe => StatMode::FIFO,
+            #[allow(unreachable_patterns)]
```

Comment:
Why not just delete the `_ => StatMode::NULL`?

Reply:
Keeping it here is because if more StatMode are added in future, there is no need add the default patterns again.
If no more StatMode will be added, delete is the cleaner way.